addons/wp/functions/fnc_findClosestTarget.sqf

```diff
@@ -16,27 +16,28 @@
  * Public: No
 */
 
-params ["_group", ["_radius", 500], ["_area", [], [[]]], ["_pos", [], [[]]]];
+params ["_group", ["_radius", 500], ["_area", [], [[]]], ["_pos", [], [[]]], ["_onlyPlayers", true]];
```

Comment:
Would set the default value of `_onlyPlayers` to false as it affects more than the modules

Reply:
No, before Only player were used. now allUnits can be used. to not break backwards compatibility it needs to be true